dll/win32/lpk/lpk.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * PROJECT:     ReactOS LPK
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Language Pack DLL.
+ * COPYRIGHT:   Copyright 2018 Baruch Rutman & the Wine Project 
+ */
+
+#include "ros_lpk.h"
+
+/*
+ * @implemented 
+ */
+BOOL WINAPI LpkExtTextOut(HDC hdc, int x, int y,
+                          UINT fuOptions, const RECT *lprc, LPCWSTR lpString,
+                          UINT uCount , const INT *lpDx, INT unknown)
+{   
+    LPWSTR lpReorderedString = (LPWSTR)lpString;
```

Comment:
Why assigning something to lpReorderedString when it seems it'll be reinitialized soon after (in line 28+)?